apps/vanilla/src/pages/blackJack/models/publisher.ts

```diff
@@ -0,0 +1,36 @@
+import { Subscriber } from '../type/subscriber';
+
+/** General class for all publushers. */
+export class Publisher<T> {
+	/** General array subscribers. */
+	protected subscribers: Subscriber<T>[] = [];
```

Comment:
Let's make this property `readonly`:
```suggestion
	protected readonly subscribers: Subscriber<T>[] = [];
```
Check other classes as well. We can apply the same change in other places.